Debugging.md

```diff
@@ -0,0 +1,70 @@
+# Debugging your Game Servers
+
+Sometimes your game servers cannot start and the Builds are showing up as Unhealthy. In this case, you'd like to see debug your game server running on PlayFab Multiplayer Servers Virtual Machine. Or, sometimes, even though your game server is running smoothly, you may need to connect to the Virtual Machine (VM) and see various statistics/logs about your game server.
+
+In order to connect to the VM hosting your game server (either Windows or Linux), you can get RDP/SSH credentials using [CreateRemoteUser](https://docs.microsoft.com/en-us/rest/api/playfab/multiplayer/multiplayerserver/createremoteuser?view=playfab-rest) API call or using the "Connect" button on playfab.com web application.
```

Comment:
>[CreateRemoteUser](https://docs.microsoft.com/en-us/rest/api/playfab/multiplayer/multiplayerserver/createremoteuser?view=playfab-rest) API call or using the "Connect" button on playfab.com web application. [](start = 120, length = 205)

I think the UX option should be first and maybe the only one mentioned as that is how I expect the majority of folks to do it. Since it is the primary method, I would also describe a bit more about how to navigate game manager to find the "connect" button.